src/main/java/com/microsoft/azure/msalwebsample/User.java

```diff
@@ -1,32 +1,32 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.microsoft.azure.msalwebsample;
-
-import lombok.Getter;
-import lombok.Setter;
-import org.json.JSONObject;
-
-/**
- * User object model
- */
-@Getter
-@Setter
-public class User {
-
-    protected String displayName;
-    protected String givenName;
-    protected String id;
-    protected String jobTitle;
-    protected String mail;
-    protected String mobilePhone;
-    protected String officeLocation;
-    protected String preferredLanguage;
-    protected String surname;
-    protected String userPrincipalName;
-
-    @Override
-    public String toString() {
-        return new JSONObject(this).toString();
-    }
-}
\ No newline at end of file
+//// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same as above - lets just get rid of this file.

Reply:
yes deleted this file too